miasm2/analysis/depgraph.py

```diff
@@ -759,15 +759,13 @@ def _follow_mem_wrapper(exprs, mem_read):
             follow.update(expr.get_r(mem_read=mem_read, cst_read=True))
         return follow, set()
 
-    @staticmethod
-    def _follow_mem(exprs):
+    def _follow_mem(self, exprs):
```

Comment:
This method depends on the class, not on the instance itself. So it should be a `classmethod`.